utils/robula.py

```diff
@@ -158,6 +158,10 @@ def __init__(self, element, document, config):
         self.element = element
         self.document = document
 
+        if config["ignored_attributes"] is not None and len(config["ignored_attributes"]) != 0:
+            self.attribute_black_list.extend(config["ignored_attributes"])
+            self.attribute_black_list = list(dict.fromkeys(self.attribute_black_list))
```

Comment:
Could be just
```suggestion
        if config.ignored_attributes:
            self.attribute_black_list.extend(config.ignored_attributes)
```
Pydantic guarantees us that `ignored_attributes` is either None, either list and `bool([]) is False` so it doesn't make sense to check the length here.

And this line changes nothing
```python
self.attribute_black_list = list(dict.fromkeys(self.attribute_black_list))
```

Reply:
`self.attribute_black_list = list(dict.fromkeys(self.attribute_black_list))`
With this line I wanted to make sure that there are no duplicates when we extend the list.

Reply:
In such cases it's better to leave comment explaining this, because as you can see, it was really not obvious :)
Can we convert `self.attribute_black_list` to set to avoid having duplicates there?

Reply:
Yes, we can do it with the set too. But in this way we will keep the order of the elements. I will add a comment.

Reply:
> But in this way we will keep the order of the elements
Does it matter?

Reply:
Well in this case it actually doesn't :)

Reply:
What I actually meant was this
```python
class RobulaPlus:
    def __init__(self, element, document, config):
        ...
        self.attribute_black_list = {
            "href",
            "jdn-hash",
             ...,
        }
        ...
        if config.ignored_attributes:
            self.attribute_black_list.update(config.ignored.attributes)
```